doc/locale/ja/LC_MESSAGES/developer.po

```diff
@@ -392,23 +402,26 @@ msgstr ""
 "\"ディレクトリーに保存します。"
 
 msgid ""
-"See reStructuredText (reST) `concepts and syntax <http://sphinx-doc.org/"
-"latest/rest.html>`_ about how to write a document in reStructuredText format."
+"See `reStructuredText (reST) <https://www.sphinx-doc.org/en/master/usage/"
+"restructuredtext/>`_ about how to write a document in reStructuredText "
+"format."
 msgstr ""
+"reStructuredText形式については、 `reStructuredText (reST) <https://www."
+"sphinx-doc.org/ja/master/usage/restructuredtext/>`_  を参照してください。"
 
 msgid "Confirm generated document"
-msgstr ""
+msgstr "生成されたドキュメントの確認"
 
 msgid "You can generate HTML files by the following command. ::"
-msgstr ""
+msgstr "以下のコマンドでHTMLファイルを生成することができます。"
```

Comment:
We shouldn't omit `::` because it's a reStructuredText's syntax. Did you get a warning without this?

```suggestion
msgstr "以下のコマンドでHTMLファイルを生成することができます。 ::"
```